### DISCUSSION TITLE
How to move a registry from one machine to another

### DISCUSSION THREAD

**Discussion by plastikfan** [verdaccio/verdaccio#2461] [Q&A] [answered]
I intend to use verdaccio for my own private projects that I'm the sole developer on. I've just installed on my local machine but before I get stuck into using this, I was wondering how would I move the registry from 1 machine to another.? The most pressing usecase would be, the rebuild of my machine. In this scenario I would like to be able to export the registry contents to a file of some kind, then import this into a new installation of verdaccio. Is this possible, or once a registry has been setup on a machine, it is there for eternity? Thanks.

**Comment by juanpicado** [comment, marked as answer]
No, it is just a folder, copy the **storage** folder (keep on mind the `.verdaccio-db` lives withing the storage folder) to another computer and ensure the config file points to that storage folder with an absolute path. Furthermore you should copy the htpassw file if you want to keep the users. I'd recommend use the same path locations.

**Reply by plastikfan** [reply]
Wow, thanks superman, I barely got chance to close my broswer tab before your answer came flying in thanks @juanpicado
